nixos/modules/services/databases/openldap.nix

```diff
@@ -47,26 +47,26 @@ in
       };
 
       user = mkOption {
-        type = types.string;
+        type = types.str;
         default = "openldap";
         description = "User account under which slapd runs.";
       };
 
       group = mkOption {
-        type = types.string;
+        type = types.str;
         default = "openldap";
         description = "Group account under which slapd runs.";
       };
 
       urlList = mkOption {
-        type = types.listOf types.string;
+        type = types.listOf types.str;
         default = [ "ldap:///" ];
         description = "URL list slapd should listen on.";
         example = [ "ldaps:///" ];
       };
 
       dataDir = mkOption {
-        type = types.string;
+        type = types.str;
```

Comment:
```suggestion
        type = types.path;
```

Reply:
Same reason for maybe not using `path` here.

Tbh, maybe `types.path` should be an alias for `types.str`

Reply:
Ah your suggestion in https://github.com/NixOS/nixpkgs/pull/66346#discussion_r312350479 sounds better